make_plant_parts_eia.py

```diff
@@ -1189,9 +1189,34 @@ def add_new_plant_name(self, part_df, part_name):
         part_df = (pd.merge(
             part_df, self.table_compiler.get_the_table('plants_eia'))
             .assign(plant_name_new=lambda x: x.plant_name_eia))
-        col = self.parts_to_ids[part_name]
-        part_df.loc[part_df[col].notnull(), 'plant_name_new'] = (
-            part_df['plant_name_new'] + " " + part_df[col].astype(str))
+        # we don't want the plant_id_eia to be part of the plant name, but all
+        # of the other parts should have their id column in the new plant name
+        if part_name != 'plant':
+            col = self.parts_to_ids[part_name]
+            part_df.loc[part_df[col].notnull(), 'plant_name_new'] = (
+                part_df['plant_name_new'] + " " + part_df[col].astype(str))
+        return part_df
+
+    @staticmethod
+    def add_record_count(part_df):
+        """
+        Add a record count for each set of plant part records in each plant.
+
+        Args:
+            part_df (pandas.DataFrame): dataframe containing records associated
+                with one plant part.
+            id_cols (list): list of identifying columns
```

Comment:
`id_cols` doesn't show up in the arguments of the function definition.

Reply:
good catch